fe/src/layouts/default.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <div>
+    <slot />
```

Comment:
Transfer `navbar ` here.
```suggestion
    <navbar />
    <slot />
```